packages/api-client/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import './style.css'
```

Comment:
I moved this import out of the individual layouts because it was importing everything three times.